README.md

```diff
@@ -56,11 +56,12 @@ Swagger has recently been added to the Trace Compass trace-server (reference imp
 1. Browse [to here][apiyaml] ([swagger][swagger]) or so to generate server's TSP.
 1. The resulting file is stored in the user's Downloads directory; e.g.: `~/Downloads/openapi.yaml`
 1. Copy `~/Downloads/openapi.yaml` to this directory.
-1. Update the latter with its license information and remove extra information: `./openapi.py`
+1. Update the latter with its license information and remove extra information by running `./openapi.py`
```

Comment:
Changes in this file should be part of a separate commit and together with the API changes